spec/beaker-http/http_spec.rb

```diff
@@ -22,50 +18,61 @@ module Http
           expect(subject.connection).to be_instance_of(Faraday::Connection)
         end
 
-        it 'sets a Beaker host object as the instance host object' do
-          expect(subject.host).to be_kind_of(Beaker::Host)
-        end
-
         it 'sets the middleware stack' do
-          default_middleware_stack =  [FaradayMiddleware::EncodeJson,
-                                       FaradayMiddleware::FollowRedirects,
-                                       Faraday::Response::RaiseError,
-                                       FaradayMiddleware::ParseJson,
-                                       Beaker::Http::FaradayBeakerLogger,
-                                       Faraday::Adapter::NetHttp]
-          expect(subject.connection.builder.handlers).to eq(default_middleware_stack)
+          expect(subject.connection.builder.handlers).to eq(HttpHelpers::DEFAULT_MIDDLEWARE_STACK)
         end
 
         it 'routes all http verbs to the connection object' do
-          http_verbs = [:get, :post, :put, :delete, :head, :path]
+          http_verbs = [:get, :post, :put, :delete, :head, :patch]
```

Comment:
Not sure how generic we want it, or what all Faraday supports so probably not a big deal, but what about head, options and connect?